main_app/migrations/0012_auto_20171226_0419.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.0 on 2017-12-26 04:19
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main_app', '0011_auto_20171225_1553'),
+    ]
+
```

Comment:
nitpick, blank line

Reply:
what do you mean?we haven't made changes in this file

Reply:
@yadneshk I cant get you. I can see this file in commit. Is this auto generated file?